src/strtree.c

```diff
@@ -326,97 +326,6 @@ static char evaluate_predicate(void* context, FuncGEOS_YpY_b* predicate_func,
   return errstate;
 }
 
-/* Query the tree based on input geometry and predicate function.
```

Comment:
This is no longer needed; the original `STRtree_query_bulk` did everything this did but for array inputs.